composer.json

```diff
@@ -2,10 +2,12 @@
   "name": "shoppingfeed/php-sdk",
   "license": "Apache-2.0",
   "minimum-stability": "stable",
+  "description": "Shopping Feed SDK to ease integration of our API",
   "require": {
     "php": ">= 5.6",
-    "guzzlehttp/guzzle": "^6.3",
-    "shoppingfeed/php-feed-generator": "1.0.0"
+    "ext-json": "*",
+    "shoppingfeed/php-feed-generator": "1.0.0",
+    "ql/uri-template": "^1.1"
   },
```

Comment:
"ext-json" : out of scope but was missing
"ql/uri-template" : librairie testé et maintenu ouvert a la discussion